benefits/core/models.py

```diff
@@ -104,12 +104,13 @@ class EligibilityVerifier(models.Model):
     auth_provider = models.ForeignKey(AuthProvider, on_delete=models.PROTECT, null=True)
     selection_label = models.TextField()
     selection_label_description = models.TextField(null=True)
-    start_title = models.TextField()
-    start_headline = models.TextField()
-    start_item_heading = models.TextField()
-    start_item_details = models.TextField()
-    start_item_secondary_details = models.TextField()
-    start_help_anchor = models.TextField()
+    start_template = models.TextField(null=True)
+    start_title = models.TextField(null=True)
+    start_headline = models.TextField(null=True)
+    start_item_heading = models.TextField(null=True)
+    start_item_details = models.TextField(null=True)
+    start_item_secondary_details = models.TextField(null=True)
+    start_help_anchor = models.TextField(null=True)
```

Comment:
Should this PR remove lines 108-113? Or would that cause a cascade of changes (and therefore is being saved for a later PR)?

Reply:
Was just prepping this 😉 https://github.com/cal-itp/benefits/pull/1488/commits/2aa9d196d64d89d351cdd68bf521ba42dec01023